source/uninstaller/main.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+#include <filesystem>
+#include <string>
+
+namespace fs = std::filesystem;
+int main(int argc, char** argv)
+{
+    if (fs::current_path().stem() != "Spelunky 2")
+    {
+        const fs::path exe_path{fs::path{argv[0]}.parent_path()};
+        if (exe_path.stem() != "Spelunky 2")
+        {
+            std::cout << "Not running from game folder, please move this executable to the game folder...\n";
```

Comment:
Ideally also for consistenct we should use fmt::print instead of std::cout, but I can change that later as well.